wizeline/project-lab
# Bug/project stage done

Bug #252  Contributor's Path: Stage appears completed after the first task done

#### What does this PR do?

On the Contributor's Path, marking a Project task as done, marks the Project Stage as completed, even if the Stage has more than one Task.

Project Stage should be marked as completed only when all Tasks of that Stage are marked as completed.

#### Where should the reviewer start?

`app/projects/components/Stages/Stages.tsx`

#### How should this be manually tested?

- The tester needs to join a project.
- Click on edit project.
- Click on Contributor's Path.
- On Any Stage, add a New Task, or two more, and update the project.
- Back to the project's page, mark all tasks of a project as completed.

Only when all tasks are marked as completed, the Header's background of the stage card should be red.

#### What are the relevant tickets?

#252 

#### Screenshots

Before 
![before](https://user-images.githubusercontent.com/98048770/162034582-70cbe01e-3ffa-4169-a31a-d605c75ffd46.png)

After
![after](https://user-images.githubusercontent.com/98048770/162034665-186858ce-ae94-498b-ba6a-5cd0c21c42de.png)
![after-II](https://user-images.githubusercontent.com/98048770/162034679-397c1564-7eed-45a2-a101-5f9298fcd2ba.png)


#### Checklist
